lib/openstudio-standards/standards/Standards.Fan.rb

```diff
@@ -73,13 +73,10 @@ def fan_adjust_pressure_rise_to_meet_fan_power(fan, target_fan_power)
     return true
   end
 
-  # Determines the fan power (W) based on flow rate,
-  # pressure rise, and total fan efficiency(impeller eff * motor eff)
+  # Determines the design fan flow (m3/s)
   #
```

Comment:
The method documentation got deleted somehow?

Reply:
Added @param fan[... to documentation.